server/chat-plugins/wifi.ts

```diff
@@ -785,196 +848,769 @@ const cmds: Chat.ChatCommands = {
-			reply = '<strong>Staff commands:</strong><br />' +
-					'- question or qg <em>User | OT | TID | Prize | Question | Answer[ | Answer2 | Answer3]</em> - Start a new question giveaway (voices can only host for themselves, staff can for all users) (Requires: + % @ # &)<br />' +
-					'- lottery or lg <em>User | OT | TID | Prize[| Number of Winners]</em> - Starts a lottery giveaway (voices can only host for themselves, staff can for all users) (Requires: + % @ # &)<br />' +
-					'- changequestion - Changes the question of a question giveaway (Requires: giveaway host)<br />' +
-					'- changeanswer - Changes the answer of a question giveaway (Requires: giveaway host)<br />' +
-					'- viewanswer - Shows the answer in a question giveaway (only to giveaway host/giver)<br />' +
-					'- ban - Temporarily bans a user from entering giveaways (Requires: % @ # &)<br />' +
-					'- end - Forcibly ends the current giveaway (Requires: % @ # &)<br />' +
-					'- count <em>Mon</em> - Displays how often a certain mon has been given away. Use <code>!giveaway count</code> to broadcast this to the entire room<br />';
-			break;
-		case 'gts':
-			this.checkCan('show', null, room);
-			reply = '<strong>GTS giveaway commands:</strong><br />' +
-					'- gts start <em>User | Amount | Summary of given mon | What to deposit | What to look for</em> - Starts a gts giveaway (Requires: % @ # &)<br />' +
-					'- gts left <em>Amount</em> - Updates the amount left for the current GTS giveaway. Without an amount specified, shows how many Pokémon are left, and who the latest winners are.<br />' +
-					'- gts sent <em>IGN</em> - Adds an ign to the list of latest winners, and updates left count accordingly.<br />' +
-					'- gts full - Signifies enough mons have been received, and will update the GTS giveaway to reflect that.<br />' +
-					'- gts end - Forcibly ends the current gts giveaway (Requires: % @ # &)<br />';
-			break;
-		case 'game':
-		case 'giveaway':
-		case 'user':
-			if (!this.runBroadcast()) return;
-			reply = '<strong>Giveaway participation commands: </strong> (start with /giveaway, except for /ga) <br />' +
-					'- guess or /ga <em>answer</em> - Guesses the answer for a question giveaway<br />' +
-					'- viewanswer - Shows the answer in a question giveaway (only to host/giver)<br />' +
-					'- remind - Shows the details of the current giveaway (can be broadcast)<br />' +
-					'- join or joinlottery - Joins a lottery giveaway<br />' +
-					'- leave or leavelottery - Leaves a lottery giveaway<br />';
-			break;
-		default:
-			if (!this.runBroadcast()) return;
-			reply = '<b>Wi-Fi room Giveaway help and info</b><br />' +
-			'- help user - shows list of participation commands<br />' +
-			'- help staff - shows giveaway staff commands (Requires: + % @ # &)<br />' +
-			'- help gts - shows gts giveaway commands (Requires: + % @ # &)';
-		}
-		this.sendReplyBox(reply);
+export const pages: Chat.PageTable = {
+	giveaways: {
+		default() {
```

Comment:
nit: consistently use `default` or `''`.

Reply:
`''` doesnt work still

Reply:
Now it does.